src/routes/safes/safes.controller.overview.spec.ts

```diff
@@ -1060,31 +1219,196 @@ describe('Safes Controller Overview (Unit)', () => {
 
       await request(app.getHttpServer())
         .get(
-          `/v1/safes?currency=${currency}&safes=${chain.chainId}:${safeInfo.address}`,
+          `/v1/safes?currency=${currency}&safes=${chain1.chainId}:${safeInfo1.address},${chain2.chainId}:${safeInfo2.address},${chain2.chainId}:${safeInfo3.address}&wallet_address=${walletAddress}`,
         )
-        .expect(404);
+        .expect(200)
+        .expect(({ body }) =>
+          expect(body).toMatchObject([
+            // safeInfo1 is omitted
+            {
+              address: {
+                value: safeInfo2.address,
+                name: null,
+                logoUri: null,
+              },
+              chainId: chain2.chainId,
+              threshold: safeInfo2.threshold,
+              owners: safeInfo2.owners.map((owner) => ({
+                value: owner,
+                name: null,
+                logoUri: null,
+              })),
+              fiatTotal: '4910.25',
+              queued: queuedTransactions2.count,
+              awaitingConfirmation: expect.any(Number),
+            },
+            {
+              address: {
+                value: safeInfo3.address,
+                name: null,
+                logoUri: null,
+              },
+              chainId: chain2.chainId,
+              threshold: safeInfo3.threshold,
+              owners: safeInfo3.owners.map((owner) => ({
+                value: owner,
+                name: null,
+                logoUri: null,
+              })),
+              fiatTotal: '5410.25',
+              queued: queuedTransactions3.count,
+              awaitingConfirmation: expect.any(Number),
+            },
+          ]),
+        );
     });
 
-    it('forwards error responses from the Transaction Service', async () => {
-      const chain = chainBuilder().with('chainId', '10').build();
-      const safeInfo = safeBuilder().build();
+    it('omits erroroneous Safes from the Transaction Service', async () => {
```

Comment:
Nit: `erroneous`

Reply:
Fixed in 56fee09f357b6eab83f577eebe3fd43ed21db7fb.